src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -703,6 +703,51 @@ public static void overrideServiceAccountDetailsIfProvided() {
     }
   }
 
+  /**
+   * Override default value of auto-detect with service account file path or json set in environment variable.
+   *
+   * Set values in below environment variable to override service account details in Wrangler connection page
+   * SERVICE_ACCOUNT_TYPE = FilePath or JSON
+   * SERVICE_ACCOUNT_FILE_PATH = file path of json file
+   * SERVICE_ACCOUNT_JSON = service account json
+   */
+  public static void overrideServiceAccountDetailsInWranglerConnectionPageIfProvided() {
```

Comment:
The only difference I see in `overrideServiceAccountDetailsIfProvided()` & `overrideServiceAccountDetailsInWranglerConnectionPageIfProvided()` is `CdfPluginPropertiesActions.enterValueInInputProperty("serviceAccountJSON", serviceAccountJSON);` otherwise there is a lot of duplicate code, please fix.

Reply:
Created a common method to override file path and used that in both methods to reduce the duplicate code.